### DISCUSSION TITLE
cannot import name 'CRS' from 'pyproj'

### DISCUSSION THREAD

**Discussion by cheukhin1024** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#53] [4.0 Exploring the data and building models]
I failed to import name 'CRS' from 'pyproj'. And I google-searched about it

[]https://stackoverflow.com/questions/59658167/cannot-import-name-crs-from-pyproj-for-using-the-osmnx-library/62958055

There are some suggested solutions, but it doesn't work for me because I am using Anaconda Jupyter Notebook to do modeling.

![螢幕擷取畫面 2021-05-04 044913](https://user-images.githubusercontent.com/70860455/117088941-87c52800-ad86-11eb-8e1a-a78b526aeb22.jpg)

**Comment by EY-glowglowglow** [comment]
@alexgleith @codeindulgence  any ideas on this? :)

**Comment by alexgleith** [comment]
Are you using Anaconda?

Because if so, I think you have library incompatibilities...

**Reply by cheukhin1024** [reply]
Yes. I am using Anaconda.
I am reading this article and looking for a solution.
[]https://www.activestate.com/resources/quick-reads/python-dependencies-everything-you-need-to-know/

If you have any suggestions, please tell me. Thanks. I haven't solved the problem yet
